HelpSource/Classes/UnitTest.schelp

```diff
@@ -210,14 +245,28 @@ code::
 )
 ::
 
+METHOD:: passed
+Register a passed test.
+
+code::
+this.passed(message: "this passed");
+::
+
 METHOD:: failed
-call failure directly.
+Register a test failure.
 
 code::
-UnitTest.new.passed(message: "this passed");
+this.failed(message: "this failed");
 ::
 
-SUBSECTION:: UnitTest (sub)class  example.
+EXAMPLES::
+
+Write tests by subclassing UnitTest, and defining a method beginning
+code::test_:: for each test.
+
+Make sure to implement the methods code::setUp:: and
+code::tearDown::. They will be called before and after every test.
```

Comment:
This advice from the original doc needs rewording IMO. Not every test class will need to override setUp and tearDown. Maybe something like: "If you implement the methods `setUp` and `tearDown` in your test class, they will be called before and after every test. This can help to eliminate repetitive code, and makes it easier to maintain your tests by ensuring that they all run under the same set of conditions."